.flowconfig

```diff
@@ -1,20 +1,50 @@
 [ignore]
+.*\.svg
+.*\.png
+.*/\.nyc_output/.*
+.*/docs/.*
+.*/node_modules/.cache/.*
+.*/node_modules/.*/tests?/.*
 .*/node_modules/@mapbox/jsonlint-lines-primitives/.*
-.*/node_modules/jsonlint-lines/.*
+.*/node_modules/@mapbox/mvt-fixtures/.*
+.*/node_modules/@mapbox/geojson-types/fixtures/.*
+.*/node_modules/@mapbox/mr-ui/.*
+.*/node_modules/@mapbox/dr-ui/.*
+.*/node_modules/@mapbox/batfish/.*
+.*/node_modules/browserify/.*
+.*/node_modules/browser-sync.*/.*
+.*/node_modules/nyc/.*
+.*/node_modules/fbjs/.*
+.*/node_modules/es5-ext/.*
+.*/node_modules/jsdom/.*
+.*/node_modules/eslint.*/.*
+.*/node_modules/highlight.*/.*
+.*/node_modules/rxjs/.*
+.*/node_modules/@?babel.*/.*
+.*/node_modules/react.*/.*
+.*/node_modules/svgo/.*
+.*/node_modules/moment/.*
+.*/node_modules/regenerate-unicode-properties/.*
+.*/node_modules/remark.*/.*
+.*/node_modules/webpack/.*
+.*/node_modules/caniuse-lite/.*
+.*/node_modules/d3.*/.*
+.*/node_modules/css-tree/.*
+.*/node_modules/lodash/.*
+.*/node_modules/fsevents/.*
+.*/node_modules/browser-sync-client/.*
+.*/node_modules/core-js.*/.*
 .*/node_modules/stylelint/.*
-.*/node_modules/unflowify/.*
-.*/node_modules/flow-coverage-report/.*
-.*/node_modules/babel-plugin-transform-react-remove-prop-types/.*
-.*/node_modules/conventional-changelog-core/.*
-.*/node_modules/htmltojsx/.*
+.*/node_modules/postcss.*/.*
+.*/node_modules/prismjs.*/.*
 .*/node_modules/documentation/.*
 .*/node_modules/module-deps/.*
-.*/node_modules/bcryptjs/.*
-.*/node_modules/@mapbox/geojson-types/fixtures/.*
 .*/test/unit/style-spec/fixture/invalidjson.input.json
-.*/test/integration/render-tests/.*
+.*/render-tests/.*
```

Comment:
The `render-tests`, `query-tests`, and `expression-tests` folders are all in `test/integration`. Can that folder be ignored, but the `integration/lib` folder be included?

Reply:
I'm not sure how `include`/`ignore` sections in Flow config interact, but I'd rather have it one way (and a more explicit one), not mixed, for clarity and less chance of accidental mistakes in the future.